src/apis/user.js

```diff
@@ -0,0 +1,53 @@
+const routes = require('./user-routes');
+
+class User {
+
+  constructor(requestHelper, routeHelper) {
+    this.requestHelper = requestHelper;
+    this.routeHelper = routeHelper;
+  }
+
+  createUser(params) {
+    const path = this.routeHelper.interpolate(routes.CREATE_USER, {
+      domainId: params.domainId
+    }, {
+      accountSetting: {
```

Comment:
This part won't work. The third parameter for the interpolate call is going to create a querystring portion of the URL. The data you have here is supposed to go in the body I believe. So you need to put this object into the `post()` call. Please look at the login call in the auth.js module to see how to submit a body.